templates/samples/commands/index.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Generated using theia-plugin-generator
+ */ 
+import * as theia from '@theia/plugin';
+
+const disposables: theia.Disposable[] = [];
+
+export function start() {
+    // Hello World command
+    const command: theia.Command = {
+        id: 'simple-command',
```

Comment:
could we have a convention for id ? like = sample-command-command (or template-command-something) and the label not being the same than Hello World to avoid conflicts.